ambry-api/src/main/java/com/github/ambry/quota/QuotaLoaderFactory.java

```diff
@@ -0,0 +1,4 @@
+package com.github.ambry.quota;
+
+public interface QuotaLoaderFactory {
```

Comment:
Add a todo? and license .

Reply:
this is not needed anymore. removing it.